SharedSources/MediaLibraryModel/AlbumModel.swift

```diff
@@ -16,6 +16,8 @@ class AlbumModel: MLBaseModel {
 
     var files = [VLCMLAlbum]()
 
+    var cellType: BaseCollectionViewCell.Type { return AudioCollectionViewCell.self }
```

Comment:
Should we consider putting another cell while the specific cell is done? It can be a pain if the app fatal Errors each time you want to change tab that doesn't have an `VLCMLMedia`.

Reply:
very true I can change that

Reply:
I just put moviewcell everywhere. I'll update the ticket